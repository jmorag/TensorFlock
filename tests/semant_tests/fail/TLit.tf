main = 1;
tlit : T<2,2>;
tlit = [[1. 2. 3.] [4.]];

tlit2: T<2,3,4>;
tlit2 = [[1. 2. 3.] [4.]];
