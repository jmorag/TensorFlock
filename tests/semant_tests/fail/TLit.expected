Fatal error: exception Failure("Invalid tensor literal")
